modern/src/theme/dimensions.js

```diff
@@ -2,5 +2,11 @@ export default {
   inputHeight: '42px',
   borderRadius: '4px',
   sidebarWidth: '28%',
-  tabletSidebarWidth: '52px' 
+  tabletSidebarWidth: '52px',
+  desktopDrawerWidth: '360px',
+  tabletDrawerWidth: '320px',
```

Comment:
Can we move screen size from prefix to suffix. This way it's easier to read. For example
```
drawerWidthDesktop
drawerWidthTablet
```

Reply:
Done.